NextcloudTalkUITests/NextcloudTalkUITests.swift

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright (c) 2023 Marcel Müller <marcel-mueller@gmx.de>
+//
+// Author Marcel Müller <marcel-mueller@gmx.de>
+//
+// GNU GPL version 3 or any later version
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+//
+
+import XCTest
+
+final class NextcloudTalkUITests: XCTestCase {
+
+    let timeoutLong: Double = 60
+    let timeoutShort: Double = 15
+    let server = "http://localhost:8080"
+    let username = "admin"
+    let password = "admin"
```

Comment:
You decided to not use env-vars? Isn't it better to have one source for these values that can be changed on the fly? What if I make my server in a different port, or use different credentials, I would have to change all tests to use them.

Reply:
For these tests, yes. Currently only UI tests are included here, so there's no need to have an app token right now as it is generated in the login process. And since this test is done from the user perspective, I don't see the need to have it in a generated file (username and password are hardcoded in the image anyway).

Reply:
There are only UI tests now, but they still need a docker image and a server, and IMO it's good to add envars now for future proofing. But ultimately the decision is up to you.